website/docs/r/synapse_sql_pool.html.markdown

```diff
@@ -0,0 +1,92 @@
+---
+subcategory: "Synapse"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_synapse_sql_pool"
+description: |-
+  Manages a Synapse Sql Pool.
+---
+
+# azurerm_synapse_sql_pool
+
+Manages a Synapse Sql Pool.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = "examplestorageacc"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+  account_kind             = "StorageV2"
+  is_hns_enabled           = "true"
+}
+
+resource "azurerm_storage_data_lake_gen2_filesystem" "example" {
+  name               = "example"
+  storage_account_id = azurerm_storage_account.example.id
+}
+
+resource "azurerm_synapse_workspace" "example" {
+  name                                 = "example"
+  resource_group_name                  = azurerm_resource_group.example.name
+  location                             = azurerm_resource_group.example.location
+  storage_data_lake_gen2_filesystem_id = azurerm_storage_data_lake_gen2_filesystem.example.id
+  sql_administrator_login              = "sqladminuser"
+  sql_administrator_login_password     = "H@Sh1CoR3!"
+}
+
+resource "azurerm_synapse_sql_pool" "example" {
+  name                 = "examplesqlpool"
+  synapse_workspace_id = azurerm_synapse_workspace.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Synapse Sql Pool. Changing this forces a new synapse SqlPool to be created.
+
+* `synapse_workspace_id` - (Required) The resource ID of Synapse Workspace. Changing this forces a new synapse SqlPool to be created.
```

Comment:
we shouldn't be using the term "Resource ID" because it's ambiguous between Terraform and Azure - and they can diverge

```suggestion
* `synapse_workspace_id` - (Required) The ID of Synapse Workspace within which this Sql Pool should be created. Changing this forces a new Synapse Sql Pool to be created.
```